pipeline/runtime/run.go

```diff
@@ -34,7 +34,9 @@ func executeRunStep(ctx context.Context, engine *engine.Engine, r *api.StartStep
 		return nil, nil, nil, fmt.Errorf("output variable should not be set for unset entrypoint or command")
 	}
 
-	outputFile := fmt.Sprintf("%s/%s.out", pipeline.SharedVolPath, step.ID)
+	outputFile := fmt.Sprintf("%s/%s.env", pipeline.SharedVolPath, step.ID)
```

Comment:
%s/%s-output.env

Reply:
So, that confusion is not present with DRONE_ENV one

Reply:
Sure